events/isc-fall-2017.md

```diff
@@ -5,27 +5,50 @@ title: 'InnerSource Commons Summit'
 
 ### September 27-29 in Naperville, IL, USA
 
-Join us for the next InnerSource Commons Summit in Naperville, USA.
+Join us for the InnerSource Commons Fall Summit 2017 in Naperville, USA.
+
+### Call For Presentations
+
+The Call For Presentations for the 2017 Fall Summit will run from 2017-06-07 through 2017-07-15. Please [submit your proposals for presentations, workshops, case studies and other content](/InnerSourceCommons/events/isc-fall-2017-cfp). 
+
+Some examples of possible content:
+
+* experience report (your story of what went well and what went wrong)
+* case study (a study of your InnerSource program or a specific aspect of it)
+* InnerSource-related tools
+* help needed! (present a key InnerSource problem you are facing and get help from the Commons)
+* workshops (e.g., pattern writing, pattern review, pattern mining, how to do X)
+* InnerSource survey results
+* InnerSource metrics
+* InnerSource Commons governance
+
+Presenters of accepted papers will be notified July 31st. Note that all presentations given at the Fall Summit, unless otherwise designated by the presenter, will be covered by [the Chatham House Rule](https://www.chathamhouse.org/about/chatham-house-rule).
 
 ### Venue
 
-This Summit is hosted by Nokia at their facility. More info TBA
+This Summit is hosted by [Nokia](http://www.nokia.com/en_int) at their facility at [1960 W. Lucent Lane, Naperville, IL 60563-1594](https://goo.gl/maps/cYHJb2jLb1m). The venue spaces there reserved for the Fall Summit include a flexible, large auditorium, four break-out rooms and an on-site cafeteria. 
+
+<img width="600" src="/InnerSourceCommons/assets/img/nokia_ihn_main.jpg"/>
+
+<img width="600" src="/InnerSourceCommons/assets/img/ihn_auditorium.jpg"/>
```

Comment:
During build time, HTML-Proofer is complaining that this image does not have an alt attribute.